curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-017.md

```diff
@@ -7,14 +7,48 @@ dashedName: part-17
 
 # --description--
 
-Now you need to link the `styles.css` file so the styles will be applied again. Next a self-closing `link` element in the `head` element. Give it a `rel` attribute value `stylesheet`, a `type` attribute value of `text/css`, and an `href` attribute value of `styles.css`.
+Now you need to link the `styles.css` file so the styles will be applied again. Nest a self-closing `link` element in the `head` element. Give it a `rel` attribute value `stylesheet`, a `type` attribute value of `text/css`, and an `href` attribute value of `styles.css`.
 
 # --hints--
 
-Test 1
+Your code should have a `link` element.
 
 ```js
+const link = $('link')[1];
+assert(link);
+```
+
+You should not change your existing `head` element. Make sure you did not delete the closing tag.
+
+```js
+assert($('head').length === 1);
+```
+
+Your `link` element should be within your `head` element.
+
+```js
+assert(code.match(/<head>[\w\W\s]*<link[\w\W\s]*>[\w\W\s]*<\/head>/i))
+```
+
```

Comment:
tests should check if link is closed properly  />

Reply:
Added a separate test to call that out specifically, and updated this test to include the closing slash.